### ISSUE TITLE
Package proposal: blobrunner64.vm

### ISSUE THREAD

**Issue by mike-hunhoff** [mandiant/VM-Packages#826]
### Package Name

blobrunner64

### Tool Name

blobrunner64

### Package type

ZIP_EXE

### Is the tool a console application?

false

### Tool's version number

0.0.5

### Category

Utilities

### Tool's authors

OALabs

### Tool's description

BlobRunner is a simple tool to quickly debug shellcode extracted during malware analysis.

### Download URL

https://github.com/OALabs/BlobRunner/releases/download/v0.0.5/blobrunner64.zip

### Download SHA256 Hash

325e3e26ccdce53cdd8b6665c7ed7d1765fc1c56cd088a5b4433593682c9f503

### Why is this tool a good addition?

This quote from [BlobRunner's README](https://github.com/OALabs/BlobRunner/blob/master/README.md) says it all:

> BlobRunner is a simple tool to quickly debug shellcode extracted during malware analysis. BlobRunner allocates memory for the target file and jumps to the base (or offset) of the allocated memory. This allows an analyst to quickly debug into extracted artifacts with minimal overhead and effort.

**Comment by mike-hunhoff** [comment]
See #825 for 32-bit version.

**Comment by Ana06** [comment]
@mike-hunhoff do we want to install both the 32 and the 64 versions?

**Comment by mike-hunhoff** [comment]
> @mike-hunhoff do we want to install both the 32 and the 64 versions?

Yeah my experience is that I've needed the 32-bit blobrunner for 32-bit shellcode and vice versa.

**Comment by Ana06** [comment]
@mike-hunhoff

> Yeah my experience is that I've needed the 32-bit blobrunner for 32-bit shellcode and vice versa.

Thanks for the info @mike-hunhoff! Then I suggest that we add a `blobrunner.vm` packages that installs both version. @mandiant/flare-vm does this sound good? 😄

**Comment by Ana06** [comment]
ok, thanks @mike-hunhoff! I think both are console app applications. Other than that the information LGTM!
